src/Api/Droplet.php

```diff
@@ -27,14 +27,24 @@ class Droplet extends AbstractApi
      */
     public function getAll()
     {
-        $droplets = $this->adapter->get(sprintf('%s/droplets?per_page=%d', self::ENDPOINT, PHP_INT_MAX));
-        $droplets = json_decode($droplets);
-
-        $this->extractMeta($droplets);
-
-        return array_map(function ($droplet) {
+        $thereAreNewResults = true;
+        $results = [];
+        $url = sprintf('%s/droplets?per_page=%d', self::ENDPOINT, PHP_INT_MAX);
```

Comment:
Well, maybe they’ll support 1000 at some time. I’d stick the pagination
value in a class `const` and set it to 1000.

On Wed, Sep 30, 2015 at 10:17 AM, Maciej Mączko <notifications@github.com>
wrote:

> In src/Api/Droplet.php
> <https://github.com/GraphStory/DigitalOceanV2/pull/1#discussion_r40798504>
> :
>
> > @@ -27,14 +27,24 @@ class Droplet extends AbstractApi
> >       */
> >      public function getAll()
> >      {
> > -        $droplets = $this->adapter->get(sprintf('%s/droplets?per_page=%d', self::ENDPOINT, PHP_INT_MAX));
> > -        $droplets = json_decode($droplets);
> > -
> > -        $this->extractMeta($droplets);
> > -
> > -        return array_map(function ($droplet) {
> > +        $thereAreNewResults = true;
> > +        $results = [];
> > +        $url = sprintf('%s/droplets?per_page=%d', self::ENDPOINT, PHP_INT_MAX);
>
> @funkatron <https://github.com/funkatron> I tried. They ignore any value
> > 200. As you can see, it is like this in the original module and it's not
> make any difference.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GraphStory/DigitalOceanV2/pull/1/files#r40798504>.
>